docs/src/extend/custom-rules.md

```diff
@@ -124,6 +124,7 @@ As the name implies, the `context` object contains information that is relevant
 The `context` object has the following properties:
 
 * `id`: (`string`) The rule ID.
+* `physicalFilename`: (`string`) When linting a file, it returns the full path of the file on disk without any code block information. When linting text, it returns the value passed to `—stdin-filename` or `<text>` if not specified.
```

Comment:
If possible, it would be nice to reword this to avoid using the word "returns" since this is not a method.

Reply:
@mdjermanovic How about this?

```suggestion
* `physicalFilename`: (`string`) When linting a file, it provides the full path of the file on disk without any code block information. When linting text, it provides the value passed to `—stdin-filename` or `<text>` if not specified.
```

Reply:
Sounds good to me!

Reply:
@mdjermanovic Updated 👍🏻